src/tactic/squeeze.lean

```diff
@@ -148,8 +159,17 @@ do some s ← get_proof_state_after (tac ff (user_args ++ simp_args)),
    pure (user_args' ++ simp_args')
 
 /-- make a `simp_arg_type` that references the name given as an argument -/
+-- this is currently very badly written - using an `expr` constructor raw is not a good idea
 meta def name.to_simp_args (n : name) : tactic simp_arg_type :=
-do e ← resolve_name' n, pure $ simp_arg_type.expr e
+pure $ simp_arg_type.expr (@expr.const ff n [])
```

Comment:
this works according to the tests; plus, as this only processes constants in the output of `simp`, it should still work, if ugly